instance/models/openedx_appserver.py

```diff
@@ -0,0 +1,259 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Instance app models - Open EdX AppServer models
+"""
+from django.conf import settings
+from django.db import models
+from django.template import loader
+from django.utils.text import slugify
+
+from pr_watch.github import get_username_list_from_team
```

Comment:
@itsjeyd I had them like that, but pylint complained that the `instance` app's imports were not grouped together.

I think I will remove the relative imports and use absolute imports for everything, which should resolve it.